tagstudio/src/core/library.py

```diff
@@ -1072,7 +1012,7 @@ def merge_dupe_entries(self):
                 )
             else:
                 sys.stdout.write(
-                    f"\r[LIBRARY] [{i}/{len(self.entries)}] Consolidating Duplicate: {(e.path + os.pathsep + e.filename)[0:]}..."
+                    f"\r[LIBRARY] [{i}/{len(self.entries)}] Consolidating Duplicate: {(e.path / e.filename)[0:]}..."
```

Comment:
> TypeError: 'PosixPath' object is not subscriptable

it needs to be converted to string before slicing it... or maybe the slice isnt even needed since it stars at the first character anyway?